predicators/ground_truth_models/spot/nsrts.py

```diff
@@ -39,6 +43,42 @@ def move_sampler(state: State, goal: Set[GroundAtom],
             if len(objs) == 3:
                 if objs[2].name == "extra_room_table":
                     return extra_room_table_offset
+            # For MoveToObjOnFloor
+            if len(objs) == 3:
+                if objs[2].name == "floor":
+                    obj = objs[1]
+                    # Get graph_nav to body frame.
```

Comment:
my hope is that the "state" of the spot interface (like stuff that changes) doesn't actually matter here. is that true? if not, I would be concerned that the spot interface "state" could be misaligned with the `state` passed into the sampler. we should keep all actually-changing stuff inside of the `state`.